src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -43,43 +43,31 @@ static void SetSchemaNameIfNotExist(char **schemaName, char *newSchemaName);
 
 /*
  * RelayEventExtendNames extends relation names in the given parse tree for
- * certain utility commands. The function more specifically extends table,
- * sequence, and index names in the parse tree by appending the given shardId;
- * thereby avoiding name collisions in the database among sharded tables. This
- * function has the side effect of extending relation names in the parse tree.
+ * certain utility commands. The function more specifically extends table and
+ * index names in the parse tree by appending the given shardId; thereby
+ * avoiding name collisions in the database among sharded tables. This function
+ * has the side effect of extending relation names in the parse tree.
  */
 void
 RelayEventExtendNames(Node *parseTree, char *schemaName, uint64 shardId)
 {
 	/* we don't extend names in extension or schema commands */
 	NodeTag nodeType = nodeTag(parseTree);
-	if (nodeType == T_CreateExtensionStmt || nodeType == T_CreateSchemaStmt)
+	if (nodeType == T_CreateExtensionStmt || nodeType == T_CreateSchemaStmt ||
+		nodeType == T_CreateSeqStmt || nodeType == T_AlterSeqStmt)
 	{
 		return;
 	}
 
 	switch (nodeType)
 	{
-		case T_AlterSeqStmt:
-		{
-			AlterSeqStmt *alterSeqStmt = (AlterSeqStmt *) parseTree;
-			char **sequenceName = &(alterSeqStmt->sequence->relname);
-			char **sequenceSchemaName = &(alterSeqStmt->sequence->schemaname);
-
-			/* prefix with schema name if it is not added already */
-			SetSchemaNameIfNotExist(sequenceSchemaName, schemaName);
-
-			AppendShardIdToName(sequenceName, shardId);
-			break;
-		}
```

Comment:
Weird.  I'd short for a second that the support for this was there to make ALTER SEQUENCE OWNED BY dist_rel; work (even though that'd not make much sense as a sequence is only owned by one rel), but it's not that.  Not sure what this could mean.  @ozgune @sumedhpathak Any input?